inst/assets/bibliography.bib

```diff
@@ -1912,7 +1912,7 @@ @article{yaverouglu2014revealing
 
 @article{Lee2019,
     author = {Lee, Young Ho},
-    title = {Strengths and Limitations of Meta-Analysis},
+    title = {{Strengths and Limitations of Meta-Analysis}},
```

Comment:
I just realized that this is problematic because the capitalization for all words depends on the style. 

It would be better to curly bracket only those words, specifically, where upper/lowercase matters. 

Sorry for noticing just now, if you can fix this for all the refs that would be great..